application/src/js/cms/rd-table-objects.js

```diff
@@ -323,65 +342,6 @@ function groupedTable(data, title, subtitle, footer,  category_column, parent_co
     };
 }
 
-// ---------------------------------
-// PREPROCESSING
```

Comment:
Maybe we should keep the "Preprocessing" version and ditch the "Common functions" version, as it's only called once (by preProcessTableObject) and doesn't seem to be common to anything else?

Reply:
Sure